DigitalLearningSolutions.Data.Tests/Services/CustomPromptsDataServiceTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.Helpers;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class CustomPromptsDataServiceTests
+    {
+        private ICustomPromptsDataService customPromptsDataService;
+
+        [SetUp]
+        public void Setup()
+        {
+            var connection = ServiceTestHelper.GetDatabaseConnection();
+            customPromptsDataService = new CustomPromptsDataService(connection);
+        }
+
+        [Test]
+        public void GetCentreCustomPromptsByCentreId()
+        {
+            // Given
+            var expectedCentreCustomPromptsResult = CustomPromptsTestHelper.GetDefaultCentreCustomPromptsResult();
+
+            //When
```

Comment:
NIP: Super minor, but can we have a space before "When" like we do for other comments?